src/renderer/components/Button/ButtonStandard.tsx

```diff
@@ -22,24 +24,30 @@ const BaseButton: React.FunctionComponent<IBaseButtonProps> = ({
     focused,
     onClick,
     buttonType,
-}) => (focused ? <button 
+    large,
+    buttonId,
+}) => (focused ? <button
+    id={buttonId}
     className={`button btn-${buttonType} focused`} 
     disabled={disabled} 
     onClick={onClick}>
-    {children}
+    <div className={`${large ? "div-large" : "" }`}>{children}</div>
 </button> : <button 
```

Comment:
don't ever duplicate code like this. Generate classes using function or something

Reply:
I'm reffering to "focus" class/prop